Gopkg.toml

```diff
@@ -80,7 +75,7 @@
 
 [[override]]
   name = "github.com/docker/cli"
-  branch = "19.03"
+  revision = "d83cd90464377d4164c8f70248d064b979e5ca98"
```

Comment:
I'll open a backport to current branches; if that's merged soon, we don't have to make this change